frontend/src/pages/projects/screens/detail/notebooks/ShowAllButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Badge, Button, Flex, FlexItem } from '@patternfly/react-core';
+import React from 'react';
+
+type ShowAllButtonProps = {
+  visibleLength: number;
+  isExpanded: boolean;
+  totalSize: number;
+  onToggle: () => void;
+};
+
+const ShowAllButton: React.FC<ShowAllButtonProps> = ({
```

Comment:
FYI, this separate button component is created  to have identical function for both cluster storage and data connection as mentioned in the mock